Mlem/App/Views/Root/Tabs/Settings/Components/SettingsHeaderView.swift

```diff
@@ -29,7 +29,7 @@ struct SettingsHeaderView<IconView: View>: View {
     
     var body: some View {
         Section {
-            VStack(spacing: 12) {
+            VStack(alignment: .leading, spacing: 12) {
                 icon
                     .symbolVariant(.fill)
                     .padding(.bottom, 5)
```

Comment:
*nit* I think this spacing is a little closer to iOS settings

```suggestion
            VStack(alignment: .leading, spacing: 4) {
icon
.symbolVariant(.fill)
.padding(.bottom, 11)
```

Reply:
Yeah, that is closer 👍